kolibri/plugins/style_guide/assets/src/views/content/radio-buttons/index.vue

```diff
@@ -1,159 +1,64 @@
 <template>
 
-  <component-template
-    class="radio-style-guide"
-    :codeExamplesTemplate="codeExamplesTemplate"
-    :api="api"
-    :requirePath="requirePath"
-  >
-
-    <h1 slot="title">Radio Buttons</h1>
-    <p slot="summary">
-      Radio buttons allow the selection of a single option from a set of 3 or
-      more options
-    </p>
-
-    <div slot="guidelines-and-usage">
-
-      <h3>Usage</h3>
-      <p>
-        In particular, radio buttons aid the user when they need to see all
-        available options side-by-side
-      </p>
-      <p>
-        If it is not necessary to see all available options, a dropdown may be
-        used
-      </p>
-      <p>
-        Selected state of controls generally denote "positives"
-        (yes/selected/etc.)
-      </p>
-
-      <h3>Do not</h3>
-      <ul>
-        <li>
-          Use radio buttons as a filter in a table or a list
-        </li>
-        <li>
-          Nest radio buttons
-        </li>
-      </ul>
-
-      <h3>Style</h3>
-      <p>
-        Kolibri utilizes the default styles + padding from Keen UI
-      </p>
-      <p>
-        Selected option is the action color, other available options are dark
-        grey
-      </p>
-      <p>
-        Where possible, avoid using icons with radio buttons. If necessary, the
-        icon should sit within the label area, pushing the label to the right.
-      </p>
-
-      <h3>Labels & option text:</h3>
-      <p>
-        Keep these short, concise, and easy for the user to understand. Should
-        not be structured as a sentence.
-      </p>
-
-      <h3>Layout</h3>
-      <p>
-        Radio buttons are always left-aligned
-      </p>
-      <p>
-        No inset padding, radio buttons left-align with the content of their
-        container
-      </p>
-      <p>
-        Radio buttons are vertically stacked
-      </p>
-    </div>
-  </component-template>
+  <page-template title="Radio Buttons" :completed="false">
+
+    <h2><code>{{ kRadioButtonApi.name }}</code> API</h2>
+    <component-docs :api="kRadioButtonApi" />
+
+    <h2>Code Example</h2>
+    <vue-example :code="example" />
+
+    <h2>Guidelines</h2>
+    <h3>Usage</h3>
+    <ul>
+      <li>Should be used when all options must be visible. If not, use a select.</li>
+      <li>Selected state of radio buttons should denote positives (yes/selected)</li>
+      <li>Should not be used to filter tables or lists</li>
+      <li>Should not be nested</li>
+    </ul>
+
+    <h3>Style</h3>
+    <ul>
+      <li>Always left-aligned</li>
+      <li>Vertically stacked</li>
+    </ul>
+
+    <h3>Language</h3>
+    <ul>
+      <li>Labels should be short and concise. Should not be sentences.</li>
+    </ul>
+
+  </page-template>
```

Comment:
Seems like good text & copy cleanup to me.

In general, let's try to do text & copy cleanup as independent PRs so we can more easily run changes by the designers.